.github/CODEOWNERS

```
@@ -1,4 +1,5 @@
 /articles/quickstart @auth0/dx-sdks-approver
 /articles/libraries @auth0/dx-sdks-approver
 /articles/cms @auth0/dx-sdks-approver
-/articles/quickstart/webapp/express @pdillon
+/articles/quickstart/webapp/nodejs @pdillon
+/articles/quickstart/webapp/nodejs @davidpatrick
```

Can these live on the same line?

```suggestion
/articles/quickstart/webapp/nodejs @pdillon @davidpatrick
```